data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+
+
+ship "Entasi"
+	sprite "ship/avgi entasi"
+	thumbnail "thumbnail/avgi entasi"
+	swizzle 0
+	attributes
+		category "Utility"
+		licenses
+			"Avgi Civilian"
+		"shields" 1500
+		"hull" 3600
+		"required crew" 6
+		"bunks" 12
+		"mass" 210
+		"drag" 0.6
+		"heat dissipation" 1.2
+		"fuel capacity" 1200
+		"cargo space" 240
+		"outfit space" 420
+		"weapon capacity" 120
+		"engine capacity" 42
+		"solar collection" 12
+		"solar heat" 3.6
+		"anchor point" 2
+		weapon
+			"blast radius" 360
+			"shield damage" 1800
+			"hull damage" 1200
+			"hit force" 1200
+	outfits
+		"Phased Array Laser (Blue)" 2
+		"Sidereal Launcher" 2
+		"Sidereal Missile" 84
+
+		"Ultracapacitor Bank" 3
+		"Diffuse Deflector"
+		"Optical Lasing Generator"
+		"Multispectral Scanner" 3
+		"Spectrometer Array" 6
+		"Electromotive Rifle" 9
+
+		"Magnetoplasma Drive" 2
+		"Hyperdrive"
+
+	gun -36 9 "Sidereal Launcher"
+	gun 36 9 "Sidereal Launcher"
+	gun -36 9
+	gun 36 9
+	turret -10 72 "Phased Array Laser (Blue)"
+	turret 10 72 "Phased Array Laser (Blue)"
+	turret 0 -4
+	bay "Fighter" 0 -30
+		over
+	bay "Fighter" 0 -30
+		under
+	bay "Drone" -18 -45
+	bay "Drone" 18 -45
+	bay "Drone" -18 -18
+	bay "Drone" 18 -18
+	bay "Drone" -18 72
+	bay "Drone" 18 72
+
+	engine -45 45
+		zoom 0.9
+	engine 45 45
+		zoom 0.9
+	engine -39 45
+		zoom 0.9
+	engine 39 45
+		zoom 0.9
+	engine -33 45
+		zoom 0.9
+	engine 33 45
+		zoom 0.9
+	engine -30 45
+		zoom 0.9
+	engine 30 45
+		zoom 0.9
+	"reverse engine" -21 51
+		zoom 0.7
+	"reverse engine" 21 51
+		zoom 0.7
+	"reverse engine" -18 51
+		zoom 0.7
+	"reverse engine" 18 51
+		zoom 0.7
+	"steering engine" 12 -69
+		zoom 0.7
+		angle 270
+		left
+	"steering engine" 12 -63
+		zoom 0.7
+		angle 270
+		left
+	"steering engine" -12 -69
+		zoom 0.7
+		angle 90
+		right
+	"steering engine" -12 -63
+		zoom 0.7
+		angle 90
+		right
+	"steering engine" 9 87
+		zoom 0.7
+		angle 270
+		right
+	"steering engine" 9 90
+		zoom 0.7
+		angle 270
+		right
+	"steering engine" -9 87
+		zoom 0.7
+		angle 90
+		left
+	"steering engine" -9 90
+		zoom 0.7
+		angle 90
+		left
+
+	leak "leak" 60 60
+	leak "flame" 60 60
+	explode "tiny explosion" 15
+	explode "small explosion" 12
+	explode "medium explosion" 9
+	"final explode" "final explosion large"
+
+	description "The Entasi is a mid-sized mining vessel powered by a pair of solar thermal generators. With the power and weapons capacity available to support laser mining weapons, the Entasi is able to quickly and efficiently break asteroids, whose mineral contents are then collected by the significant fleet of Sonikis cargo drones it can carry into an asteroid field."
+
+
+ship "Undsyni"
+	sprite "ship/avgi undsyni"
+	thumbnail "thumbnail/avgi undsyni"
+	swizzle 0
+	attributes
+		category "Utility"
+		licenses
+			"Avgi Civilian"
```

Comment:
```suggestion
		licenses
			"Avgi Civilian"
		"cost" 10400000
```
This is similar in size and HP total to the smaller human heavy warships, but with a significantly reduced armament in exchange for a larger fuel capacity and a huge cargo bay. Hence, a similar cost to the 9.2M of the Container Transport or the 9.8M of the Leviathan, but slightly higher than either due to the broader capabilities.